TODO.md

```diff
@@ -0,0 +1,19 @@
+# TODO
+- upgrade to Svelte@4
+- migrate e2e/svelte to Vite (see e2e/svelte/README.md)
+- cleanup README
+- add eslint with plugins (svelte, import, promise)
+- use different config loader https://github.com/unjs/c12
+- use different build tool https://github.com/unjs/unbuild
+- 
+- add https://github.com/TrySound/rollup-plugin-terser
```

Comment:
I prefer unminified developer tools as it makes debugging them so much easier. Plus, npm has compression anyway

Reply:
Yeah, thats true.